src/Services/QueuedJobService.php

```diff
@@ -304,6 +304,10 @@ public function queueJob(QueuedJob $job, $startAfter = null, $userId = null, $qu
      */
     public function startJob($jobDescriptor, $startAfter = null)
     {
+        if (!$this->queueHandler) {
```

Comment:
We shouldn't be doing this, a hard exception here is better if `$this->queueHandler` is falsey

`$this->queueHandler` is set via Injector as specified in [yml config](https://github.com/symbiote/silverstripe-queuedjobs/blob/4/_config/queuedjobs.yml#L12) - , by default it is [configured](https://github.com/symbiote/silverstripe-queuedjobs/blob/4/_config/queuedjobs.yml#L6) to use DefaultQueueHandler